package.json

```diff
@@ -6,6 +6,7 @@
   "typings": "fastify.d.ts",
   "scripts": {
     "lint": "standard | snazzy",
+    "checkdeps": "node -e 'c = child_process.spawnSync(\"npm\", [\"ls\"]); console.log(c.status === 0 ? \"\": c.stderr.toString()); process.exit(c.status)'",
```

Comment:
Just to be sure, will this work also under Windows?

Reply:
:smile: 
I have no idea. But I'm using nodejs without doing nothing in bash. I think so.

Anyway if only I need this "feature" we are able to not land it